rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -22,15 +22,29 @@
 {%- block body %}
 <header class="row">
 
+  {% if record | get_cover_art%}
+  <a id="thumbnail" class="col-sm-2 col-md-1 d-flex justify-content-start" href="{{ (record | get_cover_art) }}" target="_blank">
```

Comment:
Are we sure that we need an external link with the thumbnail url?

Reply:
Yes, it's asked by the PO

Reply:
Yes